src/web/components/PortalHeader/PortalHeader.tsx

```diff
@@ -36,10 +40,39 @@ export function PortalHeader({
   const { participantId } = useParams();
   const { LoggedInUser } = useContext(CurrentUserContext);
 
-  const routes = [ParticipantInformationRoute, TeamMembersRoute, EmailContactsRoute];
-  if (LoggedInUser?.user?.isUid2Support) {
-    routes.push(AuditTrailRoute);
-  }
+  const [routes, setRoutes] = useState<PortalRoute[]>([
+    ParticipantInformationRoute,
+    TeamMembersRoute,
+    EmailContactsRoute,
+  ]);
+
+  useEffect(() => {
```

Comment:
I think this `useEffect` would become unneccessary if `LoggedInUser` had all the user role info

Reply:
Yep - not needed